src/Illuminate/Support/helpers.php

```diff
@@ -298,7 +298,7 @@ function retry($times, callable $callback, $sleepMilliseconds = 0, $when = null)
 
         try {
             return $callback($attempts);
-        } catch (Exception $e) {
+        } catch (Throwable $e) {
```

Comment:
This is a breaking change, so needs to target `12.x`. Before doing anything, wait for Taylor's review.

Reply:
thanks, good to know 😄

Reply:
@taylorotwell please have a look and let us know if this should be targeted to `master`

Reply:
@sethsandaru , why not this?

```php
catch (Exception|Throwable $e) {
```

Reply:
@devajmeireles it's unnecessary to do so, `Exception` implements `Throwable` and using union, it still introduces breaking changes